// sunar/dist/utils/enums.cjs
'use strict';

// src/utils/enums.ts
var Builders = /* @__PURE__ */ ((Builders2) => {
  Builders2[Builders2["Slash"] = 0] = "Slash";
  Builders2[Builders2["ContextMenu"] = 1] = "ContextMenu";
  Builders2[Builders2["Protector"] = 2] = "Protector";
  Builders2[Builders2["Signal"] = 3] = "Signal";
  Builders2[Builders2["Button"] = 4] = "Button";
  Builders2[Builders2["Modal"] = 5] = "Modal";
  Builders2[Builders2["SelectMenu"] = 6] = "SelectMenu";
  Builders2[Builders2["Autocomplete"] = 7] = "Autocomplete";
  Builders2[Builders2["Group"] = 8] = "Group";
  return Builders2;
})(Builders || {});
var CooldownScope = /* @__PURE__ */ ((CooldownScope2) => {
  CooldownScope2[CooldownScope2["User"] = 0] = "User";
  CooldownScope2[CooldownScope2["Channel"] = 1] = "Channel";
  CooldownScope2[CooldownScope2["Guild"] = 2] = "Guild";
  CooldownScope2[CooldownScope2["Global"] = 3] = "Global";
  return CooldownScope2;
})(CooldownScope || {});

exports.Builders = Builders;
exports.CooldownScope = CooldownScope;
